x-pack/solutions/search/plugins/search_query_rules/public/components/query_ruleset_detail/query_rule_flyout/document_selector/draggable_list.tsx

```diff
@@ -35,70 +36,97 @@ export const DraggableList: React.FC<DraggableListProps> = ({
   onIndexSelectorChange,
   onIdSelectorChange,
   dragEndHandle,
+  errors,
 }) => {
   return (
-    <EuiDragDropContext onDragEnd={dragEndHandle}>
-      <EuiDroppable droppableId="queryRuleDroppable" spacing="m">
-        {isIdRule && actionIdsFields
-          ? actionIdsFields.map((doc, index) => (
-              <EuiDraggable
-                usePortal
-                spacing="m"
-                index={index}
-                hasInteractiveChildren={true}
-                draggableId={'queryRuleDocumentDraggable-' + doc + '-' + index}
-                key={doc + '-' + index}
-                isDragDisabled={pinType === 'exclude'}
-                data-test-subj={`queryRuleDocumentDraggable-id-${doc}-${index}`}
-              >
-                {() => (
-                  <DocumentSelector
-                    initialDocId={doc}
-                    indexDoc={index}
-                    type={pinType}
-                    hasIndexSelector={false}
-                    onDeleteDocument={() => {
-                      onDeleteDocument(index);
-                    }}
-                    onIdSelectorChange={(id) => {
-                      onIdSelectorChange(index, id);
-                    }}
-                  />
-                )}
-              </EuiDraggable>
-            ))
-          : actionFields.map((doc, index) => (
-              <EuiDraggable
-                usePortal
-                spacing="m"
-                index={index}
-                hasInteractiveChildren={true}
-                draggableId={'queryRuleDocumentDraggable-' + doc._id + '-' + index}
-                key={doc._id}
-                isDragDisabled={pinType === 'exclude'}
-                data-test-subj={`queryRuleDocumentDraggable-doc-${doc._id}-${index}`}
-              >
-                {() => (
-                  <DocumentSelector
-                    initialDocId={doc._id}
-                    indexDoc={index}
-                    type={pinType}
-                    index={doc._index}
-                    onDeleteDocument={() => {
-                      onDeleteDocument(index);
-                    }}
-                    onIndexSelectorChange={(indexName) => {
-                      onIndexSelectorChange(index, indexName);
+    <>
```

Comment:
is this outer `<></>` still needed? all I see inside it is `<EuiDragDropContext>`

Reply:
I think I added a callout here and removed afterwards. Not necessary.